Add Solomon Smith to Contributors list

SolomonSmith-dev opened the pull request:
Before submitting this pull request, check the changes to see it's only the changes you made intentionally
If there are changes to other lines you didn't make deliberately, it's possible that your IDE made the changes with a utility like prettier.
Next time, make sure that you only add your changes by using `git add -p` and rather than `git add Contributors.md`

If you're doing something in the checklist below, put an `x` inside `[ ]` so that `- [ ]` becomes `- [x]`

- [x] I had fun going through this tutorial (ノ^o^)ノ and learned on the way ٩(＾◡＾)۶
- [ ] There are some things I'd like to improve in this tutorial. I have written them below.
- [ ] There were steps where I had errors while following this tutorial. I have written them below.

SolomonSmith-dev committed:
Fix: remove extra line from Contributors.md

DamperDoor56 commented:
Hello! 😄 
This branch contains conflicts, try adding your name somewhere in the middle of the file, not at the beginning or the end. Just like the instructions said: [Don't add it at the beginning or end of the file. Put it anywhere in between.](https://github.com/firstcontributions/first-contributions?tab=readme-ov-file#make-necessary-changes-and-commit-those-changes)

SolomonSmith-dev commented:
Hey, I fixed the commit by updating the correct file (Contributors.md) and removed the incorrect one. Let me know if there's anything else I need to do — thanks!

DamperDoor56 reviewed (commented):
Delete the line 2525 and it will merge!

SolomonSmith-dev commented on Contributors.md:
```diff
@@ -2521,3 +2522,4 @@ Matthew Burgos
 - [cpcap](https://github.com/cpcap1214)
 - [Yassine Erradouani](https://github.com/yassnemo)
 - [Julio Florentino](https://github.com/JulioFlorentino)
+
```
All fixed — I removed the extra line at the bottom of Contributors.md as requested. Thanks for the review and guidance! And Thanks for your patience
